src/runtime/manual_tests/firebase-ng-tests.ts

```diff
@@ -60,6 +60,7 @@ describe('firebase-ng-driver', function() {
     const driver2 = new FirebaseDriver<number>(storageKey, Exists.ShouldExist);
     await driver2.init();
 
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
I think you can remove 'async' from the below function, it doesn't seem to use it.

Reply:
done.  Thanks!